site/content/demo/code-highlight.md

```diff
@@ -16,15 +15,15 @@ const ExampleCode = () => {
 }
 ```
 
-Eg. Python
+**Python example:**
```

Comment:
As a matter of markdown style I suggest using a heading instead of bolding headings.